packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -549,7 +550,7 @@ sdk.server('https://eu.api.example.com/v14');`),
               // need to clean them up.
               str = str.replace(REF_PLACEHOLDER_REGEX, '$1');
 
-              writer.writeLine(`${str} as const`);
+              writer.write(`${str} as const`);
```

Comment:
Same as the other semicolon fix. These schemas were getting generated like the following:

```ts
const schema = {"type": "string"} as const
;
```

They're now this:

```ts
const schema = {"type": "string"} as const;
```

<sub>its the little things</sub>